Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```
@@ -280,8 +282,8 @@
             Slash: 6
             Heat: 12
             Structural: 20 # "expertly contained to cut through nearly any material" yet it had no structural damage
-  - type: GuideHelp  
-    guides: [ SecurityWeapons ]  
+  - type: GuideHelp
+    guides: [ SecurityWeapons ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)